Sprint-1/implement/max.js

```diff
@@ -1,4 +1,18 @@
 function findMax(elements) {
+    if(elements.length == 0){
+        return -Infinity;
+    }else if(elements.length == 1 && typeof(elements[0])== 'number'){
```

Comment:
Why treat this case as special case? The code in the next block can also handle this case.